### PR TITLE
Bug 15: Fixed Division by zero Error

### LINKED ISSUE #15: Bug: Division by zero error on products
# CRITICAL ISSUE

When the client requests a single product, or all products, the status code is 500 and the response is always the following.

```
ZeroDivisionError at /products
division by zero
```

The most recent feature added by the team was adding an average rating key to each product. You might want to start there and see if there are any `try/catch` blocks you can implement.

### PR DESCRIPTION
**Pull request by S-L-Murphey**
Description of PR that completes issue here...

## Changes

- Changed `models/product.py` for rating conditional. avg will run if len(ratings) is > 0 else display string.

## Requests / Responses

If this PR contains code that defines a new request/response, or changes an existing one, please put the JSON representations here.

**Request**

GET `/products/1`

**Response**

HTTP/1.1 200 OK

```json
{
    "id": 1,
    "name": "Optima",
    "price": 1655.15,
    "number_sold": 0,
    "description": "2008 Kia",
    "quantity": 3,
    "created_date": "2019-05-21",
    "location": "Onguday",
    "image_path": null,
    **"average_rating": "No Ratings Yet"**
}
```

## Testing

Description of how to test code...

- [ ] Run migrations
- [ ] Run test suite
- [ ] Seed database
- [ ] GET /products/1 and note the average_rating key. Should say "No Ratings Yet"

## Related Issues

- Fixes #15

### CODE REVIEW

**Review by gwynnemeeks — CHANGES_REQUESTED** [review]
I really love what you did here, but the ticket asks for a `try/catch` block. See if you can refactor this into a `try/catch` block.

File: bangazonapi/models/product.py [hunk 1]
```diff
@@ -62,7 +62,11 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
+        #avg = total_rating / len(ratings) if len(ratings) > 0 else "No Ratings Yet"
```
Comment (heymonicakay) [inline bangazonapi/models/product.py:65]: This commented code should be removed of it's not being used.

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Nice work with the try..except block! See my comments re: removing commented code, and then you should be all set!

**Review by gwynnemeeks — CHANGES_REQUESTED** [review]
Remove commented out code

**Pull request merged**